.github/workflows/yocto-build-deploy.yml

```
@@ -563,22 +532,60 @@ jobs:
             ${{ env.DEPLOY_PATH }}/image/balena.img
             ${{ env.DEPLOY_PATH }}/balena-image.docker
 
+
+      # Separate this evaluation into its own step + output, as we use this logic in several places and its easier to manage this way
+      - name: Evaluate whether to finalize release
+        if: steps.merge-test-result.outputs.finalize || inputs.force-finalize
```

```suggestion
        if: steps.merge-test-result.outputs.finalize == 'true' || inputs.force-finalize
```
Step outputs are always strings